src/lib/components/commons/FilterPanel/FilterPanel.styled.ts

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+import { StyledButton } from '../Button/Button.styled';
+
+type IStyledFilterProps = {
+justify?: string;
+width?: string; 
+isButtonGroup?: boolean;
+}
+
+export const StyledFilterPanel = styled.div`
+            display: grid;
+            grid-column: 1/-1;
+            gap: .5rem;
+            position: absolute;
+            top:15%;
+            background-color:#ecf0f6;
+            z-index:1;
+            border-radius: 10px;
+            padding:2%;
+            box-shadow:var(--box-shadow-1);
+`;
+
+export const StyledFilterControlDiv= styled.div<IStyledFilterProps>`
+display:flex;
+flex-wrap: wrap;
+gap:2%;
+justify-content: ${({justify}) => justify ?? 'space-evenly'};
+margin:2% 0
+`;
+
+export const StyledFilterButton = styled(StyledButton)<IStyledFilterProps>`
+width: ${({width}) => width ?? '50%'};
+
+height: 35px;
```

Comment:
in general i use rem all over... so we change then change the rem size and it'll affect all the sizes... for example 3.5 rem is the same as 35px (when the root font-size set to 10px -> you can follow the GlobalStyle file to see this logic)

Reply:
got it